itomate/itomate.py

```diff
@@ -87,6 +87,10 @@ async def render_tab_panes(tab, panes, pofile_name):
         for command in pane_commands:
             await current_session.async_send_text(f"{command}\n")
 
+        # Leave a command in the prompt, awaiting execution
```

Comment:
Take a look at how the root_path command is getting inserted into the commands. https://github.com/kamranahmedse/itomate/blob/c0e00a52716da49d3039e6d01bed0888e57cf212/itomate/itomate.py#L187. Let's stick to the pattern of organizing the order of commands before the panes are being passed to be rendered. This will get rid of some of this duplicate code also.

Reply:
Thanks, that's a great suggestion. One question now though - it looks like all commands have `\n` appended during the pane render. How do you recommend we prevent this only for the last command in the array?

Reply:
Lets append the /n outside of the render when we are looping. That will solve that problem